utils/checks.stanza

```diff
@@ -483,22 +482,22 @@ defn compute-equivalent-resistance (pull-up-resistors: Tuple<Instance>) :
       val resistance = check-pull-up-resistor-does-not-have-0-ohm-resistance(resistor)
       1.0 / resistance
 
-defn check-sink-current-meets-collector-ratings (rail-voltage: Double, pull-up-resistors: Tuple<Instance>, open-collectors: Tuple<DigitalPin>) :
+defn check-sink-current-meets-collector-ratings (net-voltage: Toleranced, pull-up-resistors: Tuple<Instance>, open-collectors: Tuple<DigitalPin>) :
   val equivalent-resistance = compute-equivalent-resistance(pull-up-resistors)
   ; Next, compute the sink current (V / R) and check if it is in spec for the driver.
-  val sink-current = rail-voltage / equivalent-resistance
+  val sink-current = max-value(net-voltage) / equivalent-resistance
```

Comment:
This needs to be modified a bit?

val sink-current = (max-value(net-voltage) - min-value(vol(driver))) / equivalent-resistance

which will calculate the maximum sink-current and check it against the driver spec.